server/routes/api-v1/userRoutes.ts

```diff
@@ -7,4 +7,6 @@ userRouter.post('/login', authController.login);
 
 userRouter.post('/logged', authController.getToken);
 
+userRouter.get('/logo', authController.getApplicationLogo);
```

Comment:
This should not be in a user router. I would like to implement this under `/organization/...` path.
We will use `/organization/...` for getting users from organization.

Reply:
So you suggest making organization router and controller yes? For me this is fine

Reply:
imo the organization's logo has nothing to do with user.

>We will use /organization/... for getting users from organization.

that's wrong. Users should be available at `/users`

Reply:
Asking again. What is your suggestion?

Reply:
Make endpoint like:  GET `/organization/logo`

Reply:
And organization router?